SAK-44561 Error parsing the date/time PA SYSTEM Create Banner

MRutea opened the pull request.

adrianfish commented on pasystem/pasystem-tool/tool/src/webapp/scripts/pasystem-tool.js:
```diff
@@ -23,7 +23,6 @@ $(function () {
             // add clear action if present
             $datepicker.siblings().find(".clear-datepicker-btn").click(function() {
               $datepicker.val("");
-              $("[name='" + $datepicker.attr('id') + "_selected_datetime']").val("");
```
I wouldn't just remove this, unless you know exactly where it's used. It looks like it's the hidden field generated by the date picker. I'd play it safe and do:
const hidden = this.querySelector(`[name='${$datepicker.attr('id')}_selected_datetime']`);
hidden && hidden.value = "";

adrianfish reviewed (changes requested):
See what you think about my comment.

MRutea commented:
@adrianfish 
Hi, Adrian;

Thanks for your comment.
I think the line was no longer used, after the different tests I did.
That's why I removed it.

But you're right.
Let's play safe. I added the code you wrote for me.

Thanks again.

adrianfish reviewed (approved):
Good to go, as long as you tested my suggested change - just in case :)
